src-frontend/app/templates/signup.hbs

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-3"></div>
+    <div class="col-md-6 login-container">
+      <form class="form-horizontal" {{action 'signup' on='submit'}}>
+        <div class="form-group">
+          <label for="email" class="col-sm-2 control-label">E-Mail</label>
+          <div class="col-sm-10">
+            {{input type="text" value=email class="form-control"}}
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="username" class="col-sm-2 control-label">Username</label>
+          <div class="col-sm-10">
+            {{input type="text" value=username class="form-control"}}
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="password" class="col-sm-2 control-label">Password</label>
+          <div class="col-sm-10">
+            {{input type="password" value=password1 class="form-control"}}
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="password_confirmation" class="col-sm-2 control-label">Confirm Password</label>
+          <div class="col-sm-10">
+            {{input type="password" value=password2 class="form-control"}}
+          </div>
+        </div>
+        <div class="form-group">
+            <div class="col-sm-10 col-md-push-2">
+                <label for="accept_tos" class="control-label">
+                    {{input type="checkbox" name=accept_tos checked=accept_tos}}
+                    I have read and accept the TOS
+                </label>
+            </div>
+        </div>
+        <div class="form-group">
+          <div class="col-sm-offset-2 col-sm-10">
+            <button type="submit" class="btn btn-default">Sign up</button>
+          </div>
+        </div>
+
+        <div class="form-group">
+            {{#link-to 'login'}}Already have an account? Log in{{/link-to}}
```

Comment:
Change to `Already have an account? {{#link-to 'login'}}Click here to log in{{/link-to}}`